gammapy/conftest.py

```diff
@@ -23,7 +23,8 @@
 
 # Treat all DeprecationWarnings as exceptions
 from astropy.tests.helper import enable_deprecations_as_exceptions
-enable_deprecations_as_exceptions()
+#TODO: add numpy again once https://github.com/astropy/regions/pull/252 is addressed
+enable_deprecations_as_exceptions(warnings_to_ignore_entire_module=["numpy"])
```

Comment:
have it regions rather than numpy, I suspect you want to catch the ones triggered by numpy directly in gammapy.
also, maybe `enable_deprecations_as_exceptions(modules_to_ignore_on_import=['regions'])` would be the right incantations (I never know, and have to try out both).

Reply:
Thanks, in my previous comment I wasn't clear enough. The `DeprecationWarning` is actually raised by numpy, because of a call to `np.asscalar()` in regions. So neither `warnings_to_ignore_entire_module=["region"]` nor `modules_to_ignore_on_import=['regions'])`   ignores the error. I have submitted a fix in https://github.com/astropy/regions/pull/252.